### ISSUE TITLE
Upon mention of user not already in Banter, prompt author of post to send email invite

### ISSUE THREAD

**Issue by jeffdomke** [blockstack-radiks/banter#8]
- When I want to @ mention a specific person, but they have never used Banter
- I want to invite them to Banter
- So that they see what I posted and reply

Rational for this is that it closes a basic viral loop for adoption and retention. 

I imagine that a user authoring a post can mention any Blockstack ID, but often a certain user will not be a Banter user (no email address). Upon checking Radiks, the app can prompt the author user if they wish to invite that targeted user, simply by adding an email address. This would send an transactional invite with the text from the mention.

**Comment by jeffdomke** [comment]
Bump think we should include this.

**Comment by friedger** [comment]
Banter should not promote the notion of accounts or banter users. 

Ideally, banter can lookup the communications channels in the user's profile or banter can offer the author to share the post to the user via Web Share API.

**Comment by hstove** [comment]
I get what you're saying, but the Web Share API only works on Android, [according to MDN](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share).

It would be fantastic to have some decentralized communications tool, and this is an idea we've brainstormed about, but there are many unsolved challenges still.

I'm going to close this, as we shipped this feature already. @friedger , if you'd like to continue the conversation, feel free to open an issue!
